docs/reference/operations/create_transfers.md

```diff
@@ -0,0 +1,309 @@
+# `create_transfers`
+
+Create one or more [`Transfer`s](../transfers.md). A successfully created transfer will modify the
+amount fields of its [debit](../transfers.md#debit_account_id) and
+[credit](../transfers.md#credit_account_id) accounts.
+
+## Examples
+
+- [Asset Exchange](../../recipes/asset-exchange.md)
+
+## Event
+
+The transfer to create. See [`Transfer`](../transfers.md) for constraints.
+
+## Result
+
+Results are listed in this section in order of descending precedence — that is, if more than one
+error is applicable, only the one listed first is returned.
```

Comment:
This is just a little bit unclear that you're switching between just listing them and talking about a specific instance of a call. Maybe

> Results are listed in this section in order of descending precedence — that is, if more than one error is applicable for a transfer being created, only the one listed first error is returned.